cmd/website.go

```diff
@@ -66,7 +66,7 @@ var websiteCmd = &cobra.Command{
 		log.Debug(networkInfo.String())
 
 		// Connect to Redis
-		redis, err := datastore.NewRedisCache(redisURI, networkInfo.Name)
+		redis, err := datastore.NewRedisCache(redisURI, networkInfo.Name, redisReadonlyURI)
```

Comment:
I think we should switch the arguments, to have name first, and then all the URIs